File: CS/GridControlCRUDMVVMInfiniteAsyncSource/IssuesContext/Issue.cs
using System;

namespace GridControlCRUDMVVMInfiniteAsyncSource {
    public class Issue {
        public int Id { get; set; }
        public string Subject { get; set; }
        public int UserId { get; set; }
        public virtual User User { get; set; }
        public DateTime Created { get; set; }
        public int Votes { get; set; }
        public Priority Priority { get; set; }
    }
    public enum Priority { Low, BelowNormal, Normal, AboveNormal, High }
}

File: VB/GridControlCRUDMVVMInfiniteAsyncSource/IssuesContext/Issue.vb
Imports System

Namespace GridControlCRUDMVVMInfiniteAsyncSource
	Public Class Issue
		Public Property Id() As Integer
		Public Property Subject() As String
		Public Property UserId() As Integer
		Public Overridable Property User() As User
		Public Property Created() As DateTime
		Public Property Votes() As Integer
		Public Property Priority() As Priority
	End Class
	Public Enum Priority
		Low
		BelowNormal
		Normal
		AboveNormal
		High
	End Enum
End Namespace
